### PR TITLE
docs: Describe the release procedure

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Appium Java Client Release Procedure
+
+This document describes the process of releasing this client to the Maven repository.
+Its target auditory is project maintainers.
+
+## Release Steps
+
+1. Update the [Changelog](../CHANGELOG.md) for the given version based on previous commits.
```
Comment (SrinivasanTarget) [inline docs/release.md:8]: We also need to bump the version here https://github.com/appium/java-client/blob/master/gradle.properties

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/release.md:8]: added

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+1. Create a pull request to approve the changelog.
+1. Merge the changelog after it is approved.
+1. Bump the `appiumClient.version` number in [gradle.properties](../gradle.properties).
```
Comment (valfirst) [inline docs/release.md:11]: I guess version bump should be a part of the previous commit:

```suggestion
1. Update the [Changelog](../CHANGELOG.md) for the given version based on previous commits.
1. Bump the `appiumClient.version` number in [gradle.properties](../gradle.properties).
1. Create a pull request to approve the changelog and version bump.
1. Merge the pull request after it is approved.
```

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+1. Create and push a new repository tag. The tag name should look like 
+  `v<major_number>.<minor_number>.<patch_number>`. 
+1. Login to Nexus using 1Password credentials. Ask Appium maintainers
+  if you need access to the team's 1Password vault.
+1. Open [Sonatype](https://oss.sonatype.org/#welcome) in your browser.
+1. Navigate to `Staging Repositories`.
```
Comment (valfirst) [inline docs/release.md:17]:
```suggestion
1. Open [Sonatype](https://oss.sonatype.org/#welcome) in your browser.
1. Login to Nexus using 1Password credentials. Ask Appium maintainers
  if you need access to the team's 1Password vault.
1. Navigate to `Staging Repositories`. 
```

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  `v<major_number>.<minor_number>.<patch_number>`. 
+1. Login to Nexus using 1Password credentials. Ask Appium maintainers
+  if you need access to the team's 1Password vault.
+1. Open [Sonatype](https://oss.sonatype.org/#welcome) in your browser.
+1. Navigate to `Staging Repositories`. 
+1. Select the corresponding release and click `Close`.
+1. Wait until checks are completed.
+1. Click `Release`.
+1. Verify the new version is published in the 
+  [Maven Repository](https://mvnrepository.com/artifact/io.appium/java-client).
```
Comment (valfirst) [inline docs/release.md:22]:
```suggestion
1. After the release, the new version will be published to [Maven Central] (https://repo1.maven.org/maven2/io/appium/java-client/) within 30 minutes. Once artifacts are on Maven Central, it normally takes 1-2 hours before they appear in [search results](https://central.sonatype.com/artifact/io.appium/java-client).
```

**Review by valfirst — APPROVED** [review]

**Pull request merged**